src/test/java/api/loans/DueDateScheduledNoticesProcessingTests.java

```diff
@@ -151,6 +149,7 @@ public void uponAtNoticeShouldBeSentWhenProcessingJustAfterDueDate() {
   public void afterRecurringNoticeShouldBeSentSeveralTimesBeforeLoanIsClosed() {
 
     DateTime justAfterDueDateTime = dueDate.plusSeconds(1);
+    templateClient.create(new JsonObject().put("id", afterTemplateId.toString()));
```

Comment:
Can this be extracted to a method to better convey the intent, e.g. `createDummyTemplate`?

Reply:
And potential moved to a fixture so it can be used across test classes